incubator/binding-mqtt/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -2117,9 +2070,8 @@ private void onDecodeUnsubscribe(
             }
         }
 
-        private void sendNewSessionStateForUnsubscribe(
+        private void doSendSessionState(
             long traceId,
-            long authorization,
             List<String> topicFilters)
         {
             List<Subscription> currentState = sessionStream.getSubscriptions();
```

Comment:
We don't use `getXxx()` JavaBeans coding style, instead use `xxx()` record-style.